<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultsWindow</class>
 <widget class="QDialog" name="resultsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Results</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QLabel" name="resultsText">
     <property name="text">
      <string>&lt;Your results here&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>&amp;Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="text">
        <string>&amp;Continue</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>&amp;Stop</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
